eth/query/block.go

```diff
@@ -212,6 +215,10 @@ func GetTxObjectFromBlockResult(
 	}
 	txObj.Input = eth.EncBytes(input)
 
+	if evmAuxStore.IsDupEVMTxHash(evmTxHash) {
```

Comment:
Can just check `txObjt.Hash` at this point, can't you?

Reply:
`txObj.Hash` is encoded in Hex.

Reply:
Ok I can decode and then check